### ISSUE TITLE
Discover page causes crash

### ISSUE THREAD

**Issue by feruzm** [eSteemApp/esteem-surfer#33]
After few refresh Discover page causes crash on Windows machines

I couldn't reproduce this on Mac and Windows, 
- [ ] need double check to make sure there is no memory leak
- [ ] after few refresh list doesn't randomize anymore, use New/Created feed as well to increase randomness

**Comment by Transisto** [comment]
The installer was being auto deleted by McAfee AV.

**Comment by talhasch** [comment]
Discover page updated based on hardcoded user list. https://github.com/eSteemApp/esteem-surfer/commit/c1cc38369a6a7139117b9e99abbdc786ce93ff7c

**Comment by horpeyloaded** [comment]
@Transisto whitelist the installer in your McAfee AV so it won't get deleted again

**Comment by talhasch** [comment]
@horpeyloaded  I dont think this problem related with discover page. Maybe you can define an exception for surfer to avast?
